index.js

```diff
@@ -43,6 +44,7 @@ exports.isCI = !!(
   env.CI ||                      // Travis CI, CircleCI, Gitlab CI, Appveyor, CodeShip
   env.CONTINUOUS_INTEGRATION ||  // Travis CI
   env.BUILD_NUMBER ||            // Jenkins, TeamCity
+  env.TDDIUM ||                  // Solano CI
```

Comment:
There's no need to add `TDDIUM` to this list as it's taken care of by `exports.name` which will be set to `Solano CI` if the `TDDIUM` environment variable is present